docs/specs/clients/notify/authentication.md

```diff
@@ -118,8 +118,243 @@ A non-ideal way to avoid the race condition is for the sender to set the message
 
 ## wc_notifyDelete response
 
-- act - `notify_delete_response``
+- act - `notify_delete_response`
 - iss - did:key of dapp authentication key
 - aud - did:key of client identity key
 - app - did:web of app domain that this request is associated with 
   - Example: `did:web:app.example.com`
+
+## `wc_notifyGetNotifications`
+
+Paginated list of notifications with the most recent first. Unread notifications may be dispersed throughout the list and on subsequent pages.
+
+### Request
+
+- act - `notify_get_notifications`
+- iss - did:key of client identity key
+- ksu - key server for identity key verification
+- aud - did:key of dapp authentication key
+- app - did:web of app domain that this request is associated with 
+  - Example: `did:web:app.example.com`
+- lmt - the max number of notifications to return. Default value is 10, maximum value is 50.
+- aft - the notification ID to start returning messages after. Null to start with the most recent notification
+
+```typescript
+{
+  auth: string,
+}
+```
+
+| IRN     |              |
+| ------- | ------------ |
+| TTL     | 300          |
+| Tag     | 4014         |
+| Topic   | notify topic |
```

Comment:
These should be specified in rpc methods page instead

Reply:
See PR description, we had previously discussed/agreed that it would be better to have everything on 1 page instead of two. So making steps towards that in this PR